py/strato/common/log/log2text.py

```diff
@@ -195,8 +200,9 @@ def _process_go_logs(self, line):
                 message += '\tFile {}, line {}, in {}\n'.format(s['File'], s['Line'], s['Name'])
         if self._shouldPrintKv:
             # add key-value fields
+            extra_data.update(parsed_line)
             message += '\t'
-            message += ', '.join(['{}={}'.format(k, v) for k, v in parsed_line.iteritems() if v != ''])
+            message += ', '.join(['{}={}'.format(k, v) for k, v in extra_data.iteritems() if v != ''])
```

Comment:
didnt we want to add here request-id key-value even if self._shouldPrintKv is false?